### ISSUE TITLE
Simplified summaries reading level can't be calculated issues

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1476]
<img width="1126" height="657" alt="Image" src="https://github.com/user-attachments/assets/08b41850-3c57-443c-955c-54ef6ef41a9d" />

1. Can the "0th" message in the metabox also say that a simplified summary cannot be calculated?
2. It's also confusing that there's a message in the metabox that says, "Your simplified summary has a reading level lower than 9th grade." if it can't be calculated.
3. Why can't it calculate a reading level from a summary this long?
